src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -183,51 +183,33 @@ internal partial class CodeFixService : ICodeFixService
                 if (diagnostic.IsSuppressed)
                     continue;
 
-                cancellationToken.ThrowIfCancellationRequested();
```

Comment:
Didn't want to keep the cancellation token? More not sure if anything *else* is checking it around here.

Reply:
so in this case the change of the CT firing here is practically nill.  this is basically just taking a small list of values and converting to a dictiony (without any processing work).

i will add a CT check on all method entrypoints though.